universal-application-tool-0.0.1/app/views/AwsFileUploadStrategy.java

```diff
@@ -0,0 +1,124 @@
+package views;
+
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.each;
+import static j2html.TagCreator.form;
+import static j2html.TagCreator.input;
+import static j2html.attributes.Attr.ENCTYPE;
+
+import controllers.applicant.routes;
+import j2html.attributes.Attr;
+import j2html.tags.ContainerTag;
+import j2html.tags.Tag;
+import java.util.Optional;
+import play.i18n.Messages;
+import play.mvc.Http.HttpVerbs;
+import services.applicant.question.FileUploadQuestion;
+import services.cloud.StorageUploadRequest;
+import services.cloud.aws.SignedS3UploadRequest;
+import views.applicant.ApplicantProgramBlockEditView.Params;
+import views.questiontypes.ApplicantQuestionRendererFactory;
+import views.questiontypes.ApplicantQuestionRendererParams;
+import views.style.BaseStyles;
+import views.style.ReferenceClasses;
+import views.style.Styles;
+
+public class AwsFileUploadStrategy extends FileUploadStrategy {
```

Comment:
suggestion (blocking): `FileUploadStrategy` sounds pretty generic and both the interface and its implementations are very focused on rendering HTML. Recommend renaming:
```suggestion
public class AwsFileUploadViewStrategy extends FileUploadViewStrategy {
```

Reply:
Done